### DISCUSSION TITLE
thanhle.blog/neu-mot-viec-la-kho-khan-sao-phai-dam-dau

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#81] [Ideas]
# thanhle.blog/neu-mot-viec-la-kho-khan-sao-phai-dam-dau

Khi người ta đang lo bị đuổi việc, thì tôi tự xin nghỉ 😤

https://thanhle.blog/blog/neu-mot-viec-la-kho-khan-sao-phai-dam-dau

**Comment by khiem20tc** [comment]
Vì yêu cứ đâm đầu, yêu tiền yêu công việc yêu bản thân hay yêu cái tham vọng muốn làm cái gì đó thay đổi thế giới. Có lẽ mọi chữ "khổ" đều xuất phát từ chữ "yêu" thôi anh ạ. Đôi lúc chúng ta nên cho phép bản thân mình "vô tri hưởng thái bình" một chút, break-time để nhìn lại những thành tựu của bản thân, hi vọng vớt vát chút hạnh phúc nhỏ thôi thôi anh ạ.

Nhân tiện thì... làm lon bia tâm sự thôi anh hỉ ?! :)))

**Reply by thanhlmm** [reply]
Tối nay đi uống ko :))

**Reply by khiem20tc** [reply]
Triển thôi anh :)))

**Comment by EdwardNgo** [comment]
A ơi blog a có tẩm gì không mà em đọc phê quá. E đọc hết các bài mấy lần rồi, lâu lâu vào lại đọc
